neuromation/api/parser.py

```diff
@@ -39,6 +53,25 @@ def volume(self, volume: str) -> Volume:
             storage_uri=storage_uri, container_path=container_path, read_only=read_only
         )
 
+    def build_secret_files(self, input_volumes: Set[str]) -> Set[SecretFile]:
+        secret_files: Set[SecretFile] = set()
+        for volume in input_volumes:
+            parts = volume.split(":")
+            if len(parts) != 3:
+                raise ValueError(f"Invalid secret file specification '{volume}'")
+            container_path = parts.pop()
+            secret_uri = self.parse_secret_resource(":".join(parts))
+            secret_files.add(SecretFile(secret_uri, container_path))
+        return secret_files
+
+    def parse_secret_resource(self, uri: str) -> URL:
```

Comment:
Should the method be public or private?
What is the usage outside of the `Parser` class?

Reply:
I agree with @asvetlov that it likely should be private method. If it should be public, rename it to `secret_resource()` or `secret_uri()` or something like.